dotcom-rendering/src/components/CricketScorecard.tsx

```diff
@@ -0,0 +1,379 @@
+}: {
+	batters: BatterData[];
+	extras: Extras;
+	inningsTotals: InningsTotals;
+}) => (
+	<table
+		css={[
+			tableStyles,
+			css`
+				${until.leftCol} {
+					border-top: 0.0625rem solid
+						${palette('--football-match-list-border')};
+				}
+			`,
+		]}
+	>
+		<thead>
+			<tr>
+				<th>Batter</th>
+				<th css={hideUntilTabletStyle}></th>
+				<th>Runs</th>
+				<th>Balls</th>
+				<th css={hideUntilTabletStyle}>4s</th>
+				<th css={hideUntilTabletStyle}>6s</th>
+			</tr>
+		</thead>
+		<tbody>
+			{batters.map((batter) => (
+				<tr key={batter.name}>
+					<th scope="row">
+						<strong>{batter.name}</strong>
+						<div css={hideFromTabletStyle}>{batter.howOut}</div>
+					</th>
+					<td css={hideUntilTabletStyle}>{batter.howOut}</td>
+					<td>{batter.runs}</td>
+					<td>{batter.ballsFaced}</td>
+					<td css={hideUntilTabletStyle}>{batter.fours}</td>
+					<td css={hideUntilTabletStyle}>{batter.sixes}</td>
+				</tr>
+			))}
+			<tr
+				css={css`
+					th,
+					td {
+						border-top: 0.0625rem dashed
+							${palette('--cricket-scorecard-divider')};
+					}
+				`}
+			>
+				<th scope="row">
+					<strong>Extras</strong>
+				</th>
+				<td css={hideUntilTabletStyle}>
+					{getExtrasDescription(extras)}
+				</td>
+				<td css={hideUntilTabletStyle} colSpan={4}>
+					{inningsTotals.extras}
+				</td>
+				<td css={hideFromTabletStyle} colSpan={2}>
+					{inningsTotals.extras}
+				</td>
+			</tr>
+			<tr>
+				<th scope="row">
+					<strong>Total</strong>
+				</th>
+				<td css={hideUntilTabletStyle}>
+					<strong>for {inningsTotals.wickets}</strong>
+				</td>
+				<td>
+					<strong>{inningsTotals.runs}</strong>
+				</td>
+				<td colSpan={3} css={hideUntilTabletStyle}>
+					<strong>{inningsTotals.overs} overs</strong>
+				</td>
+				<td css={hideFromTabletStyle}></td>
+			</tr>
+		</tbody>
+	</table>
+);
+
+const FallOfWickets = ({
+	fallOfWickets,
+}: {
+	fallOfWickets: FallOfWicketData[];
+}) => (
+	<table css={tableStyles}>
+		<thead>
+			<tr>
+				<th colSpan={3}>Fall of wickets</th>
+			</tr>
+		</thead>
+		<tbody>
+			{fallOfWickets.map((fallOfWicket) => (
+				<tr key={fallOfWicket.order}>
+					<td>{fallOfWicket.order}</td>
+					<th scope="row">{fallOfWicket.name}</th>
+					<td>{fallOfWicket.runs}</td>
+				</tr>
+			))}
+		</tbody>
+	</table>
+);
+
+type Props = {
+	allInnings: InningsData[];
+	officials: string[];
+	homeTeam: CricketTeam;
+	awayTeam: CricketTeam;
+};
+
+export const CricketScorecard = ({
+	allInnings,
+	officials,
+	homeTeam,
+	awayTeam,
+}: Props) => (
+	<Stack space={9}>
+		{allInnings.map((innings) => (
+			<div css={cricketScorecardGridStyles} key={innings.description}>
+				<h2
+					css={css`
+						${textSansBold14}
+						grid-column: centre-column-start / centre-column-end;
+						border-top: 1px solid
+							${palette('--football-match-list-top-border')};
+						padding: ${space[2]}px;
+						background-color: ${palette(
+							'--football-match-list-background',
```

Comment:
This colour could probably do with a more generic name if it's being used for more than just football match lists?